CSS/components/header.css

```diff
@@ -0,0 +1,8 @@
+.header {
```

Comment:
Same here: try to find more descriptive names when applying classes. And(!) at first figure out if you need a class for an element that is already semantic HTML!